src/main/react-front-end/src/views/Transfer/EndpointAuthenticateComponent.js

```diff
@@ -154,9 +154,26 @@ export default class EndpointAuthenticateComponent extends Component {
 		
 		let colonCount = 0;
 		for(let i=0; i < url.length; colonCount+=+(':'===url[i++]));
-		// If the Url already doesn't contain the portnumber append it else no change
-		if(colonCount===1)
-			url = url + ":" + portNum;
+
+		// Find if the port is a standard port
+		let standardPort = portNum === getDefaultPortFromUri(url);
+		if(getTypeFromUri(url) === HTTP_TYPE){
+			standardPort = portNum === getDefaultPortFromUri(HTTP_TYPE) || portNum === getDefaultPortFromUri(HTTPS_TYPE);
+		}
+
+		// If the Url already doesn't contain the portnumber and portNumber isn't standard it else no change
+		if(colonCount===1 && !standardPort){
```

Comment:
Maybe it is better to do it this way:
https://stackoverflow.com/questions/25784166/how-do-i-replace-the-port-number-in-javascript

Reply:
Thanks. Implemented your suggestions!